### DISCUSSION TITLE
Hacks are due 𝗦𝗲𝗽𝘁𝗲𝗺𝗯𝗲𝗿 𝟭𝟲𝘁𝗵, 𝟭𝟭:𝟱𝟵 𝗣𝗧!

### DISCUSSION THREAD

**Discussion by multispark** [microsoft/RAG_Hack#84] [Announcements]
While our sessions are coming to an end, you still have 4 more days to hack! Make sure to submit your projects before 𝗦𝗲𝗽𝘁𝗲𝗺𝗯𝗲𝗿 𝟭𝟲𝘁𝗵, 𝟭𝟭:𝟱𝟵 𝗣𝗧 to win prizes.

Looking forward to reviewing your projects 😎

If you missed any of the sessions, go to [𝗮𝗸𝗮.𝗺𝘀/𝗿𝗮𝗴𝗵𝗮𝗰𝗸/𝘀𝘁𝗿𝗲𝗮𝗺𝘀](https://𝗮𝗸𝗮.𝗺𝘀/𝗿𝗮𝗴𝗵𝗮𝗰𝗸/𝘀𝘁𝗿𝗲𝗮𝗺𝘀) to watch the recordings and participate in the GitHub discussions.

![RAGHack Schedule](https://github.com/user-attachments/assets/4fb0852f-b0d6-4f60-8a72-29e5257cb2c4)

**Comment by KhaoticMind** [comment]
@multispark  did the time change? The deadline is 23:59 of the 16th? That is, we have ALL THE 16th (monday) to work?
Before the deadline was 16th 00, that is, we had until the 15th 23:59. 

Am I overthinking something? :)

**Reply by multispark** [reply]
Yes, you have all of the 16th.

We updated the deadline to give some more hack time!

**Comment by erincon01** [comment]
hi, I have summited our project, bot I don't know what 2 of 9 means.
can you help me?

Thanks

![image](https://github.com/user-attachments/assets/759c784d-41df-4309-aea1-88587cc680af)

**Reply by multispark** [reply]
Hello, that is tracking which technologies you selected while submitting!

**Comment by erincon01** [comment]
Ok, thanks.

**Comment by shyshin** [comment]
Hi,

When can we see the winners list for this hack ? :)

**Reply by multispark** [reply]
Our judges are hard at work. We are aiming at having submissions judged by 9/27 or later!

**Comment by multispark** [comment]
Hello everyone, thank you for your patience. We will be announcing winners on 10/7!
